### DISCUSSION TITLE
How to change the  background color of the Room load page? (note: this is not showing up in #unanswered discussions)

### DISCUSSION THREAD

**Discussion by Charlesc22** [mozilla/hubs#2904] [General Questions & Discussion] [answered]
How do I change the background color of the page that is shown when a Room loads? I've been able to configure everything else through the admin, but can't seem to find the setting for that one. I know it can be done as I've seen other Hubs cloud instances that have changed that background color.

**Comment by emclaren** [comment]
this discussion topic is not showing up in the discussions when filtered by #unanswered. the question was re-asked here: https://github.com/mozilla/hubs/discussions/2926

**Comment by misslivirose** [comment, marked as answer]
This isn't one of the standard UI elements that we surface - you will need to change this in a custom client and deploy that to a Hubs Cloud instance. It is the var(--loading-background-color); variable in the root-vars.scss file
